survey/survey_instance.json

```diff
@@ -0,0 +1,63 @@
+[
```

Comment:
please create two _instance files.

one for each shacl file with the corresponding name. We added a CI pipeline (now existing in your branch due to the merge) which checks if the instance files are conform to the shacl (see currently failed github Actions)